app/models/framework/definition/RM3710.fdl

```diff
@@ -0,0 +1,60 @@
+Framework RM3710 {
```

Comment:
Same here. Do I just assume that you've validated them all?

Reply:
Yes. But only against a rotating sample of 1000.